README.md

```diff
@@ -11,15 +11,28 @@
 
 # Cacator
 
-Cacator (с лат. — "засранец"). Маячок для выявления несанкционированных копий нашего ПО. Работает через DNS covert channels.
+[RU](./README_RU.md)
 
-Проект является глубокой переработкой [WEASEL](https://github.com/facebookarchive/WEASEL).
+Cacator, derived from Latin, meaning "defecator," sounds like "pain in the a**" in Russian.
 
-## evwsync
+This project serves as a beacon for detecting unauthorized copies of our software. It operates using DNS covert channels.
+Initially based on WEASEL, the project has undergone significant restructuring.
+
+The main benefit of using this type of communication channel is that it hides your beacon from network scanners. Scanning
+for strange DNS requests is expensive and troublesome, so almost no one does it.
+
+It is generally illegal to deploy such trackers (and INVIAN never did). However, having such a repository in your company's
+GitHub account can help instill fear in those unscrupulous and filthy competitors who have been stealing proprietary software and ML models from
+honest companies for years without getting caught.
```

Comment:
this section looks important. i think it needs to be reflected in ru version too